x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -688,7 +688,14 @@ export function registerEncryptedSavedObjects(
       { key: 'ssl', dangerouslyExposeValue: true },
       { key: 'password', dangerouslyExposeValue: true },
     ]),
-    attributesToIncludeInAAD: new Set(['ssl', 'shipper', 'allow_edit']),
+    attributesToIncludeInAAD: new Set([
+      'service_token',
+      'shipper',
+      'allow_edit',
+      'broker_ack_reliability',
+      'broker_buffer_size',
+      'channel_buffer_size',
+    ]),
```

Comment:
I was not able to generate a ingest output object that contained any of these attributes. The two types that seemed to include encrypted properties (ssl, password), where Kafka and Logstash, and these did not contain any of the AAD properties. Any advice on how to better validate this object? Are these properties still relevant? Or are they a part of the AAD only by default of not adding them to the previous exclude list?